Use Control Plane instead of Master (#18438)
kubernetes/website: content/en/docs/reference/glossary/cluster.md

date: 2019-06-15
full_link: 
short_description: >
   A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.

aka: 
tags:
- fundamental
- operation
---
A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.

<!--more-->

> The worker node(s) host the pods that are the components of the application. The Control Plane manages the worker nodes and the pods in the cluster. In production environments, it's typical to run a Control Plane across multiple computers and also to run multiple nodes, to provide fault-tolerance and high availability.

Reviewer: ```suggestion
The worker node(s) host the pods that are the components of the application. The Control Plane manages the worker nodes and the pods in the cluster. In production environments, a Control Plane usually runs across multiple computers and a cluster usually runs multiple nodes, providing fault-tolerance and high availability.
```

Mostly grammar – if it significantly changes the meaning, let's talk about it :)
